yos/copy_test.go

```diff
@@ -305,6 +309,8 @@ func TestCopySymlink(t *testing.T) {
 			if err := CopySymlink(tt.srcPath, tt.destPath); (err != nil) != tt.wantErr {
 				t.Errorf("CopySymlink() error = %v, wantErr %v", err, tt.wantErr)
 				return
+			} else {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)](https://app.codacy.com/manual/an9an63/gut/pullRequest?prid=4927519)